pkg/apis/build/v1alpha2/build_validation.go

```diff
@@ -2,13 +2,18 @@ package v1alpha2
 
 import (
 	"context"
+	"fmt"
+	"regexp"
 
+	authv1 "k8s.io/api/authentication/v1"
 	"knative.dev/pkg/apis"
 	"knative.dev/pkg/kmp"
 
 	"github.com/pivotal/kpack/pkg/apis/validate"
 )
 
+const KpackControllerServiceAccountUsername = "system:serviceaccount:kpack:controller"
```

Comment:
```suggestion
const kpackControllerServiceAccountUsername = "system:serviceaccount:kpack:controller"
```
I don't think we want to export this field?